README.md

```diff
@@ -1,2 +1,32 @@
 # SIBRFAQ
 A list of Frequently Asked Questions for SIBR: The Society for Internet Blaseball Research
+
+## What are the API endpoints?
+
+https://www.postman.com/collections/8c0e97c7444de0a125ac
+
+## Is anyone keeping track of game updates?
+
+https://github.com/iliana/blaseball-archive-scripts
+
+Iliana is running this script locally, and we hope to have it productionized soon.
+
+## What does each stat affect?
```

Comment:
Could potentially link to the live stlats viewer, or alternately point to #project-links and keep that updated with all relevant links.